packages/jest-haste-map/src/lib/dependencyExtractor.ts

```diff
@@ -85,7 +90,8 @@ export function extract(code: string): Set<string> {
   code
     .replace(BLOCK_COMMENT_RE, '')
     .replace(LINE_COMMENT_RE, '')
-    .replace(IMPORT_OR_EXPORT_RE, addDependency)
+    .replace(IMPORT_OR_EXPORT_FROM_RE, addDependency)
+    .replace(IMPORT_RE, addDependency)
```

Comment:
how does that impact performance? could we squeeze this check into a single regex pass so we don't have to parse the text once more?